src/CMakeLists.txt

```diff
@@ -46,50 +46,49 @@ endif()
-        "${CMAKE_CURRENT_LIST_DIR}/algo.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/sync.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/stream.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/option.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/device_hub.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/environment.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/error-handling.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/hw-monitor.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/image.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/image-avx.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/log.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/option.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/rs.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/types.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/backend.cpp"
-        "${CMAKE_CURRENT_LIST_DIR}/verify.c"
+        "${CMAKE_CURRENT_LIST_DIR}/sensor.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/software-device.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/source.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/stream.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/sync.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/types.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/verify.c"
 
-        "${CMAKE_CURRENT_LIST_DIR}/software-device.h"
-        "${CMAKE_CURRENT_LIST_DIR}/environment.h"
-        "${CMAKE_CURRENT_LIST_DIR}/device_hub.h"
+        "${CMAKE_CURRENT_LIST_DIR}/algo.h"
+        "${CMAKE_CURRENT_LIST_DIR}/api.h"
         "${CMAKE_CURRENT_LIST_DIR}/archive.h"
+        "${CMAKE_CURRENT_LIST_DIR}/backend.h"
         "${CMAKE_CURRENT_LIST_DIR}/concurrency.h"
         "${CMAKE_CURRENT_LIST_DIR}/context.h"
-        "${CMAKE_CURRENT_LIST_DIR}/sensor.h"
-        "${CMAKE_CURRENT_LIST_DIR}/sync.h"
-        "${CMAKE_CURRENT_LIST_DIR}/sensor.h"
-        "${CMAKE_CURRENT_LIST_DIR}/stream.h"
-        "${CMAKE_CURRENT_LIST_DIR}/algo.h"
-        "${CMAKE_CURRENT_LIST_DIR}/option.h"
-        "${CMAKE_CURRENT_LIST_DIR}/metadata.h"
-        "${CMAKE_CURRENT_LIST_DIR}/metadata-parser.h"
+        "${CMAKE_CURRENT_LIST_DIR}/device.h"
+        "${CMAKE_CURRENT_LIST_DIR}/device_hub.h"
+        "${CMAKE_CURRENT_LIST_DIR}/environment.h"
         "${CMAKE_CURRENT_LIST_DIR}/error-handling.h"
+        "${CMAKE_CURRENT_LIST_DIR}/frame-archive.h"
         "${CMAKE_CURRENT_LIST_DIR}/hw-monitor.h"
         "${CMAKE_CURRENT_LIST_DIR}/image.h"
         "${CMAKE_CURRENT_LIST_DIR}/image-avx.h"
+        "${CMAKE_CURRENT_LIST_DIR}/metadata.h"
+        "${CMAKE_CURRENT_LIST_DIR}/metadata-parser.h"
+        "${CMAKE_CURRENT_LIST_DIR}/option.h"
+        "${CMAKE_CURRENT_LIST_DIR}/sensor.h"
+        "${CMAKE_CURRENT_LIST_DIR}/software-device.h"
         "${CMAKE_CURRENT_LIST_DIR}/source.h"
-        "${CMAKE_CURRENT_LIST_DIR}/ivcam/ivcam-private.h"
+        "${CMAKE_CURRENT_LIST_DIR}/stream.h"
+        "${CMAKE_CURRENT_LIST_DIR}/sync.h"
         "${CMAKE_CURRENT_LIST_DIR}/types.h"
-        "${CMAKE_CURRENT_LIST_DIR}/backend.h"
-        "${CMAKE_CURRENT_LIST_DIR}/device.h"
-        "${CMAKE_CURRENT_LIST_DIR}/api.h"
```

Comment:
@ev-mp because you sorted this list and changed its contents at the same time, it's hard to see what changed here. I see at least `ivcam/ivcam-private.h` is now gone, which is unrelated to T265 as far as I know.

Reply:
These changes rectify inconsistencies discovered during the validation of the new TM2 FW. And they are separate from the FW update commit